lib/perl/Genome/Disk/Command/Allocation/Unarchive.t

```diff
@@ -124,9 +133,18 @@ unlink join('/', $allocation->absolute_path, 'a.out');
 # Create command object and execute it
 my $cmd = Genome::Disk::Command::Allocation::Unarchive->create(
     allocations => [$allocation],
-    lab => 'Mardis-Wilson',
+    analysis_project => $analysis_project,
 );
 ok($cmd, 'created unarchive command');
+throws_ok(sub { $cmd->execute }, qr/currently not handled/, 'command fails with unsupported owner');
+
+$allocation->owner_id($build->id);
+$allocation->owner_class_name($build->class);
+$cmd = Genome::Disk::Command::Allocation::Unarchive->create(
+    allocations => [$allocation],
+    analysis_project => $analysis_project,
+);
+
 ok($cmd->execute, 'successfully executed unarchive command');
 is($allocation->volume->id, $volume->id, 'allocation moved to active volume');
 ok($allocation->is_archived == 0, 'allocation is not archived');
```

Comment:
Sure, it'd be a nice thing to test!  I'll have to make a testcase with an SR as the owner, since we're not making extra associations for models.